src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.props

```diff
@@ -63,4 +65,15 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
     <FscToolPath Condition=" '$(FscToolPath)' == '' ">$(MSBuildThisFileDirectory)</FscToolPath>
     <FscToolExe Condition="'$(OS)' == 'Unix' and '$(FscToolExe)' == ''">RunFsc.sh</FscToolExe>
   </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.ValueTuple" Version="$(DefaultValueTuplePackageVersion)" >
+      <DefaultInclude>true</DefaultInclude>
+    </PackageReference>
+
+    <PackageReference Include="FSharp.Core" Version="$(DefaultFSharpPackageVersion)">
```

Comment:
why not:
- put `DefaultFSharpPackageVersion` property inside .props
- put the `PAcakgeReference` inside .targets (if `DefaultFSharpPackageVersion` defined)?

why a different mechanism than `FSharpCoreImplicitPackageVersion`? is pretty much the same, the default.

like that you dont need to add and remove, just disable with a property if i dont want the default (or check if already exists i dont add that)